pkgs/tools/security/libmodsecurity/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, lib, fetchFromGitHub, autoreconfHook, pkgconfig
+, doxygen, perl, valgrind
+, curl, libxml2, pcre, lua, yajl, geoip, lmdb }:
+
+stdenv.mkDerivation rec {
+  name = "libmodsecurity-${version}";
+  version = "3.0.0-beta";
+
+  src = fetchFromGitHub {
+    owner = "SpiderLabs";
+    repo = "ModSecurity";
+    fetchSubmodules = true;
+    rev = "d465c2f1a39f0e7f6a61a1eed43c28deed98b6a9";
+    sha256 = "13xdvsksv0mijpc3l7752lg143ysp8q2cwaq2nbmvzzwdxi8flg6";
+  };
+
+  nativeBuildInputs = [ autoreconfHook pkgconfig ];
+  buildInputs = [ doxygen perl valgrind curl libxml2 pcre lua yajl geoip lmdb ];
+  configureFlags = [
+    "--enable-static"
+    "--with-curl=${curl.dev}"
+    "--with-libxml=${libxml2.dev}"
+    "--with-pcre=${pcre.dev}"
+    "--with-yajl=${yajl}"
+  ];
+
+  meta = with stdenv.lib; {
+    description = ''
```

Comment:
This is more appropriate as a `longDescription`. The `description` should be only a few words at most (so that it renders nicely in a package listing).

Reply:
Description fixed.